scripts/jenkins/ardana/ansible/roles/ses_ardana_integration/tasks/ses_configure_v1.yml

```diff
@@ -0,0 +1,31 @@
+#
+# (c) Copyright 2018 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+
+- name: Copy ceph keyring files
+  copy:
+    dest: "{{ ses_ardana_config_path }}/ceph.{{ item.item.name }}.keyring"
+    content: "{{ item.content | b64decode }}"
+    mode: 0644
+  when: "item.item.name != 'client.rgw.ses-' ~ ardana_env"
+  loop: "{{ hostvars['ses-' + ses_cluster_id].ceph_files.results }}"
+  loop_control:
+    label: "{{ item.item.name }}"
+
+- name: Generate ses-config.yml
+  template:
+    src: ses-config.yml.j2
```

Comment:
probably should name this template ses-config-v1.yml.j2

Reply:
Yes, please follow up in a new PR.